.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# .github/dependabot.yml
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'weekly'
+    commit-message:
+      prefix: chore(deps)
+      prefix-development: chore(deps)
```

Comment:
A single PR that updates both dev and prod dependencies would be fine. For this repo, updates either affect runtime dependencies of `next-intl` or anything else (docs, examples, etc.). I don't think dependabot can make a distinction between these two.